README.md

```diff
@@ -82,6 +82,29 @@ import '../../../../../test-tools/setup';
 
 Keywords from the testing frameworks can then be imported to your test files from `utils.ts`.
 
+### Functional End-to-End Testing
+
+Functional tests are run with [Testcafe](https://devexpress.github.io/testcafe/), which runs tests for each browser environment configured. The following browsers are currently tested:
+
+- Chrome
+- Safari
+
+To run the tests for all browsers together, run the following command:
+```sh
+yarn functional:test:all
+```
+
+- To run the tests for chrome only:
+```sh
+yarn functional:test:chrome
+```
+
+- To run the tests for safari only:
+```sh
+yarn functional:test:safari
```

Comment:
Comparing to an earlier command,
```
yarn test:interaction
```
the notation thus far is to start with `test`, follow it up with the type of testing (eg. `:interaction`), and then follow that up with anything more specific (eg. `:all`, `:chrome`, etc.). Let's be consistent with our previous test commands.

Reply:
Changed